src/sprites/entity.py

```diff
@@ -76,6 +79,19 @@ def get_facing_direction(self):
             self.facing_direction = 'right' if self.direction.x > 0 else 'left'
         if self.direction.y:
             self.facing_direction = 'down' if self.direction.y > 0 else 'up'
+        match self.facing_direction:
```

Comment:
if you could add a method to separate roles updating haxe_hitbox and updating face_direction

Reply:
I did this to avoid having to rewrite Player.update, but that's a good idea